etc/profile-m-z/quodlibet.profile

```diff
@@ -0,0 +1,51 @@
+# Firejail profile for quodlibet
+# Description: Music player and music library manager
+# This file is overwritten after every install/update
+# Persistent local customizations
+include quodlibet.local
+# Persistent global definitions
+include globals.local
+
+noblacklist ${MUSIC}
+noblacklist ${HOME}/.cache/quodlibet
+noblacklist ${HOME}/.config/quodlibet
+noblacklist ${HOME}/.quodlibet
+
+whitelist ${MUSIC}
+whitelist ${HOME}/.cache/quodlibet
+whitelist ${HOME}/.config/quodlibet
+whitelist ${HOME}/.quodlibet
+
+# Allow python (blacklisted by disable-interpreters.inc)
+include allow-python2.inc
+include allow-python3.inc
+
+include disable-common.inc
+include disable-devel.inc
+include disable-exec.inc
+include disable-interpreters.inc
+include disable-passwdmgr.inc
+include disable-programs.inc
+include disable-xdg.inc
```

Comment:
Does it need write access or can we `include disable-write-mnt.inc` and `read-only ${HOME}`?

Reply:
read-only ${HOME} break extensions.

Reply:
Does it work if you add `read-write ${HOME}/.quodlibet`?

Reply:
I didn't try, but extensions modify the files in the MUSIC directory.

Reply:
If it need to modify files, remove the `read-only ${HOME}`. We have already `whitelist`ing.